cfpb/cfgov-refresh
# Set up front end for reg commenting

Short description explaining the high-level reason for the pull request

## Additions

- reg-comment organism HTML and CSS
- reg-comment organism made available to main content area on sublanding pages

## Removals

- Reg comment form test page

## Changes

- Updates `@btn-font-size` variable to match Design Manual standard of 16px

## Testing

1. Pull this fork down
2. **If you didn't test #1194:**
 - Copy the new env vars to your `.env` file
 - Replace `<api_key>` with the actual API key
 - `cd` out and back into `cfgov-refresh` to activate the new env vars
 - `pip install -r requirements/local.txt`
3. `./cfgov/manage.py migrate`
4. `./cfgov/manage.py runserver`
5. Log into Wagtail running locally
6. Create a new sublanding page; give it a title
7. Add the Reg Comment module to the content area
8. Enter the test Document ID: `CFPB_FRDOC_0001-0439`
9. Publish to WWW
10. View the page, test the form
11. See result page 🎊 

## Review

- @kurtw 
- @rosskarchner 
- @richaagarwal 
- @kave 
- @sebworks 
- @amymok 

## Screenshots
![screen shot 2016-04-25 at 15 22 40](https://cloud.githubusercontent.com/assets/1044670/14822852/3bb4245a-0b9e-11e6-9089-d89a30f1c54b.png)

## Todo (not in this PR but after launch)
- Add a deadline for comments to the Wagtail module, set the form to close automatically after the deadline.

## Checklist
